### DISCUSSION TITLE
Localization from remote service

### DISCUSSION THREAD

**Discussion by johny-snow** [abpframework/abp#20204] [Q&A]
I have a microservice app. We use communication between services with AbpRemoteServiceOptions like here https://docs.abp.io/en/abp/latest/API/Dynamic-CSharp-API-Clients
But I want to get the localization resources by endpoint "/api/abp/application-localization?cultureName=en"
How can I implement this? Please help!

**Comment by maliming** [comment]
hi

I think you can inject the `IAbpApplicationLocalizationAppService` to get the `localization resources `.

In which layer/project/service do you want to get it?
